pkg/kubelet/network/kubenet/kubenet_linux.go

```diff
@@ -72,7 +72,9 @@ func (plugin *kubenetNetworkPlugin) Init(host network.Host) error {
 		glog.Warningf("Failed to find default bridge MTU: %v", err)
 	}
 
-	return nil
+	// This will load br-netfilter module and set bridge-nf-call-iptables=1
+	// to ensure that kube proxy functions correctly.
+	return plugin.NoopNetworkPlugin.Init(host)
```

Comment:
bridge-nf-call-iptables=1 will backfire when we use kubenet for anything other than a bridge plugin. Currently it should be OK since we are using bridge, please add a TODO:
```
// TODO: bridg-nf-call-iptables=1 is only useful for bridge network plugins, which kubenet 
// currently uses, but this should really be triggered based on the underlying CNI plugin type.
```